yjit.rb

```diff
@@ -60,6 +60,9 @@ def _print_stats
 
       $stderr.puts("***YJIT: Printing runtime counters from yjit.rb***")
 
+      $stderr.puts "Number of Bindings Allocated: %d\n" % counters[:binding_allocation_count]
+      $stderr.puts "Number of Local Variable Set: %d\n" % counters[:local_variable_set_count]
```

Comment:
Also change this to "Number of locals modified through binding". Otherwise it's confusing, because `setlocal` is setting local variables.

Reply:
Do we care to distinguish between local variable values being overwritten versus new local variables?  Right now this just tracks any local variable set (not whether or not there was already an existing value)

Reply:
I think for now it's ok to not distinguish. Anything that messes with the layout of the stack frame or changes local types could be problematic for us.

Reply:
Cool.  Text is changed! 😄